scenario/RewardsScenario.ts

```diff
@@ -21,7 +21,7 @@ function calculateRewardsOwed(
 scenario(
   'Comet#rewards > can claim supply rewards for self',
   {
-    filter: async (ctx) => await isRewardSupported(ctx),
+    filter: async (ctx) => await isRewardSupported(ctx) && !matchesDeployment(ctx, [{network: 'mainnet', deployment: 'weth'}]),
```

Comment:
We should add an XXX to remove this filter once rewards are supported on the WETH deployment.

Reply:
i guess alternatively, we could delete `rewardTokenAddress` from the `mainnet/WETH` configuration so that the `isRewardSupported` check works as intended

Reply:
deleted `rewardTokenAddress` from the configuration file, and deleted this filter